application/src/main/java/org/thingsboard/server/service/ws/DefaultWebSocketService.java

```diff
@@ -684,6 +685,7 @@ public void onSuccess(List<TsKvEntry> data) {
                         .queryTs(queryTs)
```

Comment:
Please extract subscription creation to a method, it is completely duplicated (except for allKeys)